### ISSUE TITLE
AbstractArray support

### ISSUE THREAD

**Issue by aml5600** [kaarthiksundar/Dubins.jl#4]
Would it be possible to extend the common input type `::Vector{Float64}` to a more general `::AbstractArray`? Would permit the use of `StaticArrays`, among other types. Additionally, switching from `Float64` to a more general type would also be beneficial.

**Comment by kaarthiksundar** [comment]
@aml5600 Yes, it is very much possible. I can get to this only end of the month. If you want the support before that, feel free to start a PR. 

Thanks

**Comment by dev10110** [comment]
addressed by https://github.com/kaarthiksundar/Dubins.jl/pull/5
